src/swoop/api/routers/jobs.py

```diff
@@ -1,43 +1,138 @@
 from __future__ import annotations
 from datetime import datetime
-
-
-from fastapi import APIRouter, Path, Query
+from pydantic import create_model
+from asyncpg import Record
+from buildpg import render
+from fastapi import APIRouter, Path, Query, Request, Depends, HTTPException
 
 from ..models import (
     Exception as APIException,
     JobList,
+    Link,
     Results,
     StatusInfo,
     InlineResponse200,
 )
+import logging
 
-
+logger = logging.getLogger("uvicorn")
+logger.setLevel(logging.INFO)
 DEFAULT_JOB_LIMIT = 1000
+DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
 
 router: APIRouter = APIRouter(
     tags=["Jobs"],
 )
 
 
+### Query Param Mapping Guide
+#
+# Job requests contain params that may not directly map to a DB field.
+# Below is an explanation of what each query param maps to.
+#
+# A join between the 'action' and 'thread' tables is necessary before filtering
+# by certain params, and generating a StatusInfo response object.
+#
+#   process_id -> action.action_name
+#   start_datetime -> event.event_time = {start_datetime}, and event.status = 'RUNNING'
+#   end_datetime -> event.event_time = {end_datetime}, and event.status = 'COMPLETED'
+#   parent_id -> action.parent_uuid
+#   job_id -> action.action_uuid
```

Comment:
@hectormachin I am uncertain of the value of this comment. 1) it appears inaccurate, and 2) with your new query being self-documenting it seems redundant.

Do you agree?